packages/keymaps/src/browser/style/index.css

```diff
@@ -126,19 +122,19 @@
 }
 
 .kb table .th-label {
-    width: 25%;
+    width: 20%;
 }
 
 .kb table .th-keybinding {
-    width: 20%;
```

Comment:
I'd maybe go 15%-20%, some keybindings are not visible enough.
In general I think that `commands` and `keybindings` should be the main priority.

Other columns can have their content shortened and can be fully displayed by hovering over the cells.

For example, look at the following case:

<img width="1251" alt="Screen Shot 2020-02-03 at 4 46 28 PM" src="https://user-images.githubusercontent.com/40359487/73693948-230faf00-46a5-11ea-9a80-6021b3137915.png">

Reply:
Update: 
1. Increased widths for commands and keybindings
2. Source is now displayed properly, the way it is done in VSCode
3. The context is resized to a smaller size to give more priority to the other two important ones. 

Ready for Review.